jenkins/master/Dockerfile

```diff
@@ -36,6 +32,8 @@ RUN mkdir -p /etc/opendevstack \
     && echo "  \"sonarqubeVersion\": \"$SONAR_VERSION\"" >> /etc/opendevstack/config.json \
     && echo "}" >> /etc/opendevstack/config.json
 
+RUN chmod g+w $JAVA_HOME/lib/security/cacerts
```

Comment:
Were you able to verify that this works both RedHat and CentOS based? It looks right to me, but would be good to verify.

Reply:
Yes, works.